src/NHSD.BuyingCatalogue.Solutions.Persistence/Repositories/SolutionRepository.cs

```diff
@@ -76,7 +85,8 @@ public async Task UpdateSupplierStatusAsync(IUpdateSolutionSupplierStatusRequest
         /// <returns>True if it exists</returns>
         public async Task<bool> CheckExists(string id, CancellationToken cancellationToken)
         {
-            var solutionCount = await _dbConnector.QueryAsync<int>(doesSolutionExist, cancellationToken, new {
+            var solutionCount = await _dbConnector.QueryAsync<int>(doesSolutionExist, cancellationToken, new
+            {
                 id
             }).ConfigureAwait(false);
```

Comment:
Since we're already ignoring line-wrapping suggestions (the lines are already too long) I think it would actually read better if it was all on one line.

I'll not insist on this one, its entirely up to you 😄

Reply:
I've moved them to all be on separate lines, as per the discussion in the code standards meeting 😄